From eb7f2f29f91b73f05ffb11a5d90ca49e0049d4ca Mon Sep 17 00:00:00 2001
From: Andreas Cord-Landwehr <cordlandwehr@kde.org>
Date: Fri, 22 Dec 2023 18:01:44 +0100
Subject: [PATCH] Remove duplicate files from install

Apparently newer qmake does not like multiple installation of same
files. Remove separate installs where parent directory already contains
the files.

Upstream-Status: Pending
Signed-off-by: Andreas Cord-Landwehr <cordlandwehr@kde.org>
---
 qmltermwidget.pro | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/qmltermwidget.pro b/qmltermwidget.pro
index c9594a9..bee7a93 100644
--- a/qmltermwidget.pro
+++ b/qmltermwidget.pro
@@ -51,15 +51,11 @@ qmldir.path += $$INSTALL_DIR/$$PLUGIN_IMPORT_PATH
 
 colorschemes.files = $$PWD/lib/color-schemes/*
 colorschemes.path = $$INSTALL_DIR/$$PLUGIN_IMPORT_PATH/color-schemes
-colorschemes2.files = $$PWD/lib/color-schemes/historic/*
-colorschemes2.path = $$INSTALL_DIR/$$PLUGIN_IMPORT_PATH/color-schemes/historic
 
 kblayouts.files = $$PWD/lib/kb-layouts/*
 kblayouts.path = $$INSTALL_DIR/$$PLUGIN_IMPORT_PATH/kb-layouts
-kblayouts2.files = $$PWD/lib/kb-layouts/historic/*
-kblayouts2.path = $$INSTALL_DIR/$$PLUGIN_IMPORT_PATH/kb-layouts/historic
 
 scrollbar.files = $$PWD/src/QMLTermScrollbar.qml
 scrollbar.path = $$INSTALL_DIR/$$PLUGIN_IMPORT_PATH
 
-INSTALLS += target qmldir assets colorschemes colorschemes2 kblayouts kblayouts2 scrollbar
+INSTALLS += target qmldir assets colorschemes
-- 
2.40.1

